src/index.js

```diff
@@ -16,6 +16,7 @@ render(
       <BrowserRouter>
         <Route path="/store" component={StoreLayout} />
         <Route path="/dev" component={DevPage} />
+        <Redirect to="/store" />
```

Comment:
any reason you wouldn't do `<Route exact "/" component={StoreLayout} />`?

Reply:
This has the added bonus of capturing any route that does not exist. 

/store/non-existant-item
/applesauce

Both of those would fall through the matches and get redirect to the main store page.